### PR TITLE
[lex-id] Wordlist updated with frequency

### PR DESCRIPTION
**Pull request by bennylin**
I've updated the tsv file with word frequency for better result.

### CODE REVIEW

**Comment by keyman-server** [comment]
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

File: release/bennylin/bennylin.id.kamus_indonesia/source/welcome.htm [hunk 1]
```diff
@@ -14,9 +14,10 @@
 <h1>Start Using Kamus Indonesia</h1>
 
 <p>
-    Kamus Indonesia 1.0 generated from template.
+    Kamus Indonesia (Indonesian dictionary) of 112519 terms, 47569 of them with word frequency from https://id.wiktionary.org/wiki/Indeks:Bahasa_Indonesia/Frekuensi/1
```
Comment (darcywong00) [inline release/bennylin/bennylin.id.kamus_indonesia/source/welcome.htm]:
```suggestion
    Kamus Indonesia (Indonesian dictionary) of 112519 terms, 47569 of them with word frequency from 
    <a href="https://id.wiktionary.org/wiki/Indeks:Bahasa_Indonesia/Frekuensi/1">https://id.wiktionary.org/wiki/Indeks:Bahasa_Indonesia/Frekuensi/1</a>
```

File: release/bennylin/bennylin.id.kamus_indonesia/source/readme.htm [hunk 2]
```diff
@@ -14,7 +14,7 @@
 <h1>Kamus Indonesia</h1>
 
 <p>
-    Kamus Indonesia 1.0 generated from template.
+    Kamus Indonesia (Indonesian dictionary) of 112519 terms, 47569 of them with word frequency from https://id.wiktionary.org/wiki/Indeks:Bahasa_Indonesia/Frekuensi/1
```
Comment (darcywong00) [inline release/bennylin/bennylin.id.kamus_indonesia/source/readme.htm]:
```suggestion
    Kamus Indonesia (Indonesian dictionary) of 112519 terms, 47569 of them with word frequency from 
    <a href="https://id.wiktionary.org/wiki/Indeks:Bahasa_Indonesia/Frekuensi/1">https://id.wiktionary.org/wiki/Indeks:Bahasa_Indonesia/Frekuensi/1</a>
```

**Comment by darcywong00** [comment]
At the moment, we don't have the infrastructure to deprecate the old name `benny_lin` to use `bennylin`. 

We might need to manually remove the old one and publish `bennylin` as version 1.0

**Comment by darcywong00** [comment]
Hi @bennylin  - we'll manually remove the old benny_lin project when this PR gets merged.

Can you go through wordlist.tsv and remove some duplicate entries?
I sorted and saw rows like:
```
aau   
aau   
ab   
ab 12
```

**Comment by darcywong00** [comment]
I'll go ahead and merge this PR, and remove the duplicate word entries on a separate PR.

**Comment by mcdurdin** [comment]
Hi @bennylin what's your status on this PR?

Note that we'll shortly be releasing an [update to the compiler](keymanapp/keyman#3338) which automatically merges duplicate entries in wordlists.

**Comment by DavidLRowe** [comment]
@bennylin Just checking on the status of this pull request. As I recall, you had some hardware issues that have delayed your contributions. If you plan to work on it still, we can leave it open. At this point, however, your PR will be out-of-date with respect to the keymanapp/lexical-models repository, so you may prefer to close this PR and start a new one.

**Comment by bennylin** [comment]
I don't have a date yet, David, Darcy, Marc. Go ahead and close it, I will reopen one later on. Thanks!

___________________

Regards,
benny 

    On Tuesday, January 26, 2021, 12:46:55 AM GMT+7, DavidLRowe <notifications@github.com> wrote:  
 
 


@bennylin Just checking on the status of this pull request. As I recall, you had some hardware issues that have delayed your contributions. If you plan to work on it still, we can leave it open. At this point, however, your PR will be out-of-date with respect to the keymanapp/lexical-models repository, so you may prefer to close this PR and start a new one.

**Comment by DavidLRowe** [comment]
Author will submit a new PR at a later date.

**Pull request closed without merging**